.github/workflows/ios.yml

```diff
@@ -1,173 +1,249 @@
+          IOS_STORE_PROFILE: ${{ secrets.IOS_STORE_PROFILE }}
+          IOS_ADHOC_PROFILE: ${{ secrets.IOS_ADHOC_PROFILE }}
+          IOS_PROFILE_KEY: ${{ secrets.IOS_PROFILE_KEY }}
+          IS_PROD: ${{ contains('production next', env.BRANCH) }}
+
+      - name: Cache node modules
+        uses: actions/cache@v1
+        with:
+          path: node_modules
+          key: ${{ runner.OS }}-npm-cache-${{ hashFiles('**/package-lock.json') }}
+          restore-keys: |
+            ${{ runner.OS }}-npm-cache-
+      - name: Install npm dependencies
+        run: |
+          npm ci
+      - name: Cache Pods dependencies
+        uses: actions/cache@v1
+        with:
+          path: ios/Pods
+          key: ${{ runner.OS }}-pods-cache-${{ hashFiles('**/ios/Podfile.lock') }}
+          restore-keys: |
+            ${{ runner.OS }}-pods-cache-
+      - name: Install pod dependencies
+        run: |
+          cd ios
+          pod install
+        shell: bash
+
+      - name: Setup google services file
+        run: echo "${{ secrets.IOS_GOOGLE_SERVICES}}" | base64 --decode > ./ios/GoogleService-Info.plist
+
+      - name: Generate assets
+        run: node ./animationAssets.js
+
+      - name: set version
+        run: |
+          BUILD_VERSION=`node -pe "require('./package.json')['version']"`
+          echo "setting version ${BUILD_VERSION}.${GITHUB_RUN_NUMBER}"
+          plutil -replace CFBundleVersion -string ${BUILD_VERSION}.${GITHUB_RUN_NUMBER} ./ios/GoodDollar/Info.plist
+          plutil -replace CFBundleShortVersionString -string ${BUILD_VERSION} ./ios/GoodDollar/Info.plist
+
+      - name: Build app
+        env:
+          ENVFILE: '../.env.${{ env.ENV }}'
+          IS_PROD: ${{ contains('production next', env.BRANCH) }}
+        run: |
+          yarn lingui:compile
+          echo "Creating release using env: ${ENVFILE}, is_prod: ${IS_PROD}"
+          PROFILE_UID="b29b41c9-e82a-4aab-96a8-e5d00e55756e"
+          if ${{ env.IS_PROD }}; then
+            PROFILE_UID="21f28d4b-7641-4420-875f-989a0bcc3d52"
+          fi
+
+          cd ios && xcodebuild archive \
+            -workspace GoodDollar.xcworkspace \
+            -scheme GoodDollar \
+            -sdk iphoneos \
+            -configuration Release \
+            -archivePath $PWD/build/GoodDollar.xcarchive \
+            PROVISIONING_PROFILE=$PROFILE_UID \
+            CODE_SIGN_IDENTITY="Apple Distribution: GoodDollar Limited (97K3Q6WTLM)"
+      - name: Export app
+        run: |
+          cd ios && xcodebuild \
+            -exportArchive \
+            -archivePath $PWD/build/GoodDollar.xcarchive \
+            -exportOptionsPlist $PWD/ci.plist \
+            -exportPath $PWD/build
+      - name: Deploy to App Center
+        run: |
+          token=${{ secrets.APPCENTER_IOS_DEV }}
+
+          if [ ${{ env.ENV }} == 'prod' ]
+          then
+            echo "setting appcenter production"
+            token=${{secrets.APPCENTER_IOS_PROD}}
```

Comment:
set this token in pre-checks - env is xxxx steps, remove the whole code block till "echo depl,oying"

Reply:
setted, removed